GithubHandler.py

```diff
@@ -144,11 +159,27 @@ def get_pr_deltas(self, pr):
         for f in file_changes:
             file_full_content = self.get_file_contents(f.filename, ref=pr.head.sha)
             full_file_delta = self.get_full_file_delta(f.filename, file_full_content)
-            file_delta = self.get_file_delta(f.patch, f.filename)
-            files_and_deltas += full_file_delta + "\n" + file_delta + "\n"
+            file_delta_str = self.get_file_delta(f.patch, f.filename)
+            files_and_deltas += full_file_delta + "\n" + file_delta_str + "\n"
 
         return files_and_deltas
 
+    def get_pr_deltas_in_list(self, pr):
```

Comment:
I'm smelling some spaghetti code. A method named `get_pr_deltas_in_list` should do what it says. But it's more tangled than my grandma's yarn. Plus, that name... Brevity's a virtue, so is clarity. Consider something like `get_patch_blocks` or `split_patch_into_blocks` for a function name that doesn’t need a map and compass.